drivers/docker/driver.go

```diff
@@ -760,6 +760,12 @@ func (d *Driver) createContainerConfig(task *drivers.TaskConfig, driverConfig *T
 		}
 		hostConfig.Runtime = d.config.GPURuntimeName
 	}
+	if driverConfig.Runtime != "" && driverConfig.Runtime != hostConfig.Runtime {
+		if hostConfig.Runtime != "" {
+			return c, fmt.Errorf("runtime '%s' requested conflicts with gpu runtime '%s'", driverConfig.Runtime, hostConfig.Runtime)
```

Comment:
what GPU runtime?  I don't understand this error message :)

Reply:
the runtime is set above as part of the `docker.nvidia_runtime` docker client configuration. looks like this errors instead of allowing a job to override the value with a configured runtime